.github/workflows/package-main-release.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with:
-          node-version: 14
+          node-version: 15
```

Comment:
This comment is here only for documentation-purposes: a few days ago we have decided to build against the newest version of node, which might not always be the LTS. You're welcome, future generations!

Reply:
To switch this version was only a test, because it stopped working somehow here https://github.com/dlr-eoc/ukis-frontend-libraries/pull/90/commits/eccac770335bf0d9b8aba7df9bd25ced22195ad3 where only patch versions are changed. Thought it could be something with the cache of  `actions/setup-node@v2`. At first I tried the newest node version 16 but it didn't work either. It looked like a problem with `node-sass` and node but it also doesn't worked after removal.

On the next week I used node 15 and it suddenly worked... I hope GitHub Actions will get better to debug in the future :)

Reply:
Hm! Well, whatever works. Thanks for the heads up!